frontend/src/test/scala/bloop/bsp/BspCompileSpec.scala

```diff
@@ -426,9 +426,9 @@ class BspCompileSpec(
       }
 
       // Change the semanticdb jar every time we upgrade Scala version
-      assert(BuildInfo.scalaVersion == "2.12.8")
+      assert(BuildInfo.scalaVersion == "2.12.12")
       val sourceDir = workspace.resolve("a").resolve("src")
-      val semanticdbJar = unsafeGetResource("semanticdb_2.12.8-4.1.11.jar")
+      val semanticdbJar = unsafeGetResource("semanticdb_2.12.12-4.1.11.jar")
```

Comment:
Does it even exist? I think that version might not be supported.

Reply:
yea exactly, I'm playing around with that now and trying to bump it, but trying to ensure we don't have any references to other versions of semantic db elsewhere. I don't want a mismatch.